File: 2.0/csharp/calls/getCallBroadcast.cs
using CallfireApiClient;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var broadcast = client.CallBroadcastsApi.Get(379506003, "name,status,labels");
    }
}

File: 2.0/java/calls/getCallBroadcast.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CallBroadcast;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        CallBroadcast broadcast = client.callBroadcastsApi().get(379506003L, "name,status,labels");
    }
}
